### ISSUE TITLE
Make uint64 support permanently enabled

### ISSUE THREAD

**Issue by e-dard** [influxdata/influxdb#15711]
In 2.0 we should have full `uint64` support enabled. This issue is to track cleaning up the technical debt left from when we had `uint64`  behind a build flag.

Start by removing this:

https://github.com/influxdata/influxdb/blob/d33ea6eea7607a4a6865fb824539204b286228f6/models/points.go#L70

And go from there...

By the end, there should be no option to enable/disable `uint64` support throughout the codebase.

**Comment by nwneisen** [comment]
I'm interested in working on this but it looks like @ayang64 has already made the change.  Is this PR just needing a signed CLA and verification that tests are passing?

**Comment by e-dard** [comment]
@nwneisen yep, Ayan took that one. It's great to see interest in contributing from the community. There is another issue here labelled as a good starter here: https://github.com/influxdata/influxdb/issues/15571

What sorts of things are you interested in within the code-base?

**Comment by nwneisen** [comment]
@e-dard I had started taking a look at #15571 after commenting.

At the moment, I'm interested in working in areas that would make me better at using InfluxDB. I answered some questions on stack overflow for Telegraf and there were some for InfluxDB that I thought I would try to answer also. 

I was going to take a look at issues tagged with `difficulty/low` next.  If there are any areas you view as higher priority, let me know and I will go for them first.
